doc/nrf/config_and_build/configuring_app/cmake/index.rst

```diff
@@ -57,33 +57,42 @@ The |NCS| uses the same CMake build variables as Zephyr and they are compatible
 For the complete list of build variables in Zephyr and more information about them, see :ref:`zephyr:important-build-vars` in the Zephyr documentation.
 The following table lists the most common ones used in the |NCS|:
 
-.. list-table:: Build system variables in the |NCS|
+.. list-table:: Common build system variables in the |NCS|
    :header-rows: 1
 
    * - Variable
      - Purpose
      - CMake argument to use
    * - Name of the Kconfig option
-     - Set the given Kconfig option to a specific value :ref:`for a single build <configuration_temporary_change_single_build>`.
+     - | Set the given Kconfig option to a specific value :ref:`for a single build <configuration_temporary_change_single_build>`.
+       | The Kconfig option name can be subject to :ref:`variable namespacing <zephyr:sysbuild_kconfig_namespacing>` and :ref:`sysbuild Kconfig namespacing <zephyr:sysbuild_kconfig_namespacing>`.
      - ``-D<name_of_Kconfig_option>=<value>``
+   * - :makevar:`CONF_FILE`
+     - | Select the base Kconfig configuration file to be used for your application and override the :ref:`autoselection process <zephyr:initial-conf>`.
+       | This variable has also been used to select one of the available :ref:`build types <modifying_build_types>`, if the application or sample supports any.
+       | Using this variable for build type selection is deprecated and is being gradually replaced by :makevar:`FILE_SUFFIX`, but :ref:`still required for some applications <modifying_build_types>`.
```

Comment:
why gradually... I would just say replaced

Reply:
Also I would add that if FILE_SUFFIX is used build_types are ignored. This is documented somewhere but can't remember where

Reply:
I will remove "gradually" when all apps and samples moved to use FILE_SUFFIXes. Currently, Matter sample and app docs, nRF Desktop, and nRF Machine Learning are still mentioning build types.